### PR TITLE
Feat: Lists

### PR DESCRIPTION
**Pull request by thesudhanshupanda**

### CODE REVIEW

File: CSS/Components/list.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  }
+  .list li:hover {
+    color: var(--primary-color);
+    cursor: pointer;
+  }
+  .stackedlist li {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+
```
Comment (sakleprajwal) [inline CSS/Components/list.css:27]: Avoid extra spaces

**Review by sakleprajwal — APPROVED** [review]
Suggested a minor change. Everything else looks good.

File: CSS/Components/list.css (hunk 1 above)
Comment (thesudhanshupanda) [reply CSS/Components/list.css:27]: alright thanks

File: CSS/Components/list.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    font-weight: 400;
+  }
+  .list li:hover {
+    color: var(--primary-color);
+    cursor: pointer;
+  }
+  .stackedlist li {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
```
Comment (Sameony) [inline CSS/Components/list.css:26]: do create utility classes for flex and justify combinations, as it'll help in upcoming projects

**Review by Sameony — APPROVED** [review]
Concise and clear code, you can merge it. 
do not shy away from creating and using utility classes

File: CSS/Components/list.css (hunk 2 above)
Comment (thesudhanshupanda) [reply CSS/Components/list.css:26]: noted Sam !

**Pull request merged**